### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by filariow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,24 @@
-# Primaza
+# :knot: Primaza
 
-## Multi Cluster Service Consumption Framework
+Primaza is a multi-cluster Service Consumption Framework.
+Primaza wants to be namespace-scoped and will not required any resource at cluster level.
```
Comment (dperaza4dustbit) [inline README.md:4]: suggest to change this one: Primaza is namespace-scoped and does not require any resource at cluster level.

**Review by dperaza4dustbit — COMMENTED** [review]
suggested a change but look good

**Review by sadlerap — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sadlerap) [reply README.md]: We definitely require CRDs, which are cluster scoped, and our webhooks are (currently) cluster-scoped as well.  Maybe we should mention this?

Comment (filariow) [reply README.md]: Added mention to CRDs. We also already have a [story for removing webhooks](https://issues.redhat.com/browse/APPSVC-1363), so I'd say we can omit them here.

Comment (dperaza4dustbit) [reply README.md]: look good to me

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Pull request merged**